lib/ditcher.js

```diff
@@ -590,6 +590,9 @@ Ditcher.prototype.doImport = function (params, cb) {
     var collectionsToImportArray = Object.keys(collectionsToImport);
 
     collectionsToImportArray.forEach(function(colKey){
+      if (collectionsToImport[colKey].length === 0) {
+        return cb(new Error("collection " + colKey + " is empty"));
```

Comment:
The outer `´forEach` will still continue after this. What if there is more than one collection and only one of them is empty? Do we want to throw an error or import the non-empty ones?